source/extensions/filters/http/cache/cache_filter_config.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <functional>
+
+#include "envoy/extensions/filters/http/cache/v3/cache.pb.h"
+
+#include "source/common/common/logger.h"
+#include "source/extensions/filters/http/cache/http_cache.h"
+#include "source/extensions/filters/http/common/pass_through_filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+using CacheFilterConfigPb = envoy::extensions::filters::http::cache::v3::CacheConfig;
+using CacheFilterConfigPbRef = std::reference_wrapper<const CacheFilterConfigPb>;
+
+// Config contains raw proto config and cache reference which is used in current filter.
+// Initially 'CacheRouteFilterConfig' is created from global config and can be
+// overriden with route config (see config.cc)
```

Comment:
This comment isn't true any more (not a cache reference, and CacheRouteFilterConfig is not created from global config). Please correct it.